### PR TITLE
Update homepage header

### PR DESCRIPTION
**Pull request by patricksimonian**
## Summary
Redid the home page header and refactor many components so that they are more usable.
## Notable Changes 
- index page layout moved to a fluid container so that it is no longer a fixed width
- renamed component PrimaryFilter to Navbar as it will become the actual navbar soon

### CODE REVIEW

File: app-web/src/components/Page/Title.module.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  margin-bottom: .5em;
+}
+
+.Title h4 {
+  font-size: 1em;
+  max-width: 525px;
+  margin: 0 auto;
+  padding: 3px;
+  line-height: 1.5em;
+}
```
Comment (ShellyXueHan) [inline app-web/src/components/Page/Title.module.css]: new line

File: app-web/src/components/Search/SearchFeedback/SearchFeedback.js [hunk 2]
```diff
@@ -24,12 +24,7 @@ const SearchFeedback = ({ searchCount, totalNodeCount, searchWordLength, query }
   let caption = null;
   // if search results returned nothing and there was a query made
   if (searchCount === 0 && query !== null) {
-    caption = (
-      <p>
-        No resources found :( Click <Link to={path}>here</Link> to reset or otherwise try searching
-        again.
-      </p>
-    );
+    caption = <p>No resources found :( Try otherwise try searching again.</p>;
```
Comment (ShellyXueHan) [inline app-web/src/components/Search/SearchFeedback/SearchFeedback.js]: <p>No resources found :( Try searching again.<p>

File: app-web/src/components/UI/Input/input.module.css [hunk 3]
```diff
@@ -25,3 +25,15 @@
...
+.Input {
+  border: 2px solid #38598a;
+  border-radius: 3px;
+  padding: 5px;
+}
+
+.Input:focus {
+  box-shadow: 0 0 2px 1px #38598aad;
+  outline: none;
+}
```
Comment (ShellyXueHan) [inline app-web/src/components/UI/Input/input.module.css]: new line

File: app-web/src/hoc/Layout.js [hunk 4]
```diff
@@ -2,7 +2,9 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 // stylesheets
-import '../assets/styles/index.css';
+// import '../assets/styles/index.css';
```
Comment (ShellyXueHan) [inline app-web/src/hoc/Layout.js]: not need the css file?

File: app-web/src/pages/index.js [hunk 5]
```diff
@@ -119,35 +115,35 @@ export class Index extends Component {
...
                 <Loading message="Loading..." />
-              ) : (
+              ) : searchResultsLength > 0 ? (
                 <Element name={REACT_SCROLL.ELEMENTS.CARDS_CONTAINER}>
                   <div className={styles.CardContainer}>
                     <Flag name="features.githubResourceCards">{SiphonResources}</Flag>
                   </div>
                 </Element>
+              ) : (
+                <p>No resources found :( try search again.</p>
```
Comment (ShellyXueHan) [inline app-web/src/pages/index.js]: use constant error message

File: app-web/src/components/Page/Title.module.css (hunk 1 above)
Comment (patricksimonian) [reply app-web/src/components/Page/Title.module.css]: fixed

File: app-web/src/components/Search/SearchFeedback/SearchFeedback.js (hunk 2 above)
Comment (patricksimonian) [reply app-web/src/components/Search/SearchFeedback/SearchFeedback.js]: oh wow...

File: app-web/src/components/Page/Title.module.css (hunk 1 above)
Comment (patricksimonian) [reply app-web/src/components/Page/Title.module.css]: fixed

**Pull request merged**